remotespark/utils/eventshandler.py

```diff
@@ -0,0 +1,15 @@
+from remotespark.utils.log import Log
+
+
+class EventsHandler:
+    def __init__(self):
+        self.logger = Log("EventsHandler")
+
+    def store_event(self, **kwargs):
```

Comment:
Why `**kwargs` rather than `**args`?

Mohamed, remember that you have to find a way to order the dimensions (so keyword arguments don't work)